igvc_navigation/src/mapper/mapper.cpp

```diff
@@ -56,7 +57,12 @@ Mapper::Mapper(ros::NodeHandle& pNh) : ground_plane_{ 0, 0, 1, 0 }
   octomapper_->create_octree(camera_map_pair_);
 
   camera_line_pub_ = pNh.advertise<pcl::PointCloud<pcl::PointXYZ>>("/mapper/camera_lines", 1);
-  camera_projection_pub_ = pNh.advertise<pcl::PointCloud<pcl::PointXYZ>>("/mapper/camera_projection", 1);
+
+  camera_projection_pub_l_ = pNh.advertise<pcl::PointCloud<pcl::PointXYZ>>("/mapper/camera_projection_l", 1);
```

Comment:
Expand these topic names out to `camera_projection_left`, etc. for clarity

Reply:
sure.